lib/versions/update.js

```diff
@@ -0,0 +1,56 @@
+const request = require('request-promise-native');
+const config = require('config');
+const { prompt } = require('enquirer');
+const promptOpts = require('../prompts');
+
+exports.desc = 'Update an existing version for your project';
+exports.category = 'services';
+exports.weight = 4;
+exports.action = 'versions:update';
+
+exports.run = async function({ opts }) {
+  const { key, version, codename, newVersion, main, beta, isPublic, deprecated } = opts;
+
+  if (!key) {
```

Comment:
There is a lot of duplicate validation for key and version here. Can we make one and refer to that so we can change the error message more easily if we need to?

Reply:
Working on a fix for this, i'll keep you updated.

Reply:
Checkout https://github.com/readmeio/rdme/pull/44